Functions/QualityControl/xASL_qc_CollectParameters.m

```diff
@@ -1,22 +1,23 @@
-function x = xASL_qc_CollectParameters(x, iSubject, ScanType)
+function x = xASL_qc_CollectParameters(x, iSubject, ScanType, iSession)
 %xASL_qc_CollectParameters Collect all parameters from structural & ASL, spread over the derivative folders
 %
-% FORMAT: x = xASL_qc_CollectParameters(x, iSubject, ScanType, CollectQCFunction)
+% FORMAT: x = xASL_qc_CollectParameters(x, iSubject, ScanType, CollectQCFunction [, iSession])
 %
 % INPUT:
 %   x                   - structure containing fields with all information required to run this submodule (REQUIRED)
 %   iSubject            - index of current subject (REQUIRED)
 %   ScanType            - string for ScanType, options = 'Structural' 'ASL' 'func' 'dwi' (REQUIRED)
+%   iSession            - index of current session (OPTIONAL, default 1, but RECOMMENDED for ScanType == 'ASL')
 %
 % OUTPUT:
 %   x                   - same as input
 %
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
-% DESCRIPTION: This function collects QC parameters for a module.
+% DESCRIPTION: This function collects QC parameters for a module for a given subject and for ASL for a given subject+session
```

Comment:
Not only for ASL, at least I would do this also for func & dwi

Reply:
Yes. Will do later.

Reply:
Done in 0b7a798c